frontend/src/shared/components/organizations-dropdown/organizations-dropdown.component.ts

```diff
@@ -0,0 +1,32 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { BehaviorSubject } from 'rxjs';
+
+@Component({
+  selector: 'tasque-organizations-dropdown',
+  templateUrl: './organizations-dropdown.component.html',
+  styleUrls: ['./organizations-dropdown.component.sass']
+})
+export class OrganizationsDropdownComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit(): void {
+  }
+
+  private selectedOrganization$ = new BehaviorSubject<string>(this.selectedOrganization);
```

Comment:
Move this to a separate service and use it from there

Reply:
Done